src/main/thrift/parquet.thrift

```diff
@@ -257,7 +257,11 @@ struct Statistics {
     */
    1: optional binary max;
    2: optional binary min;
-   /** count of null value in the column */
+   /** 
+    * count of null value in the column 
+    *
+    * Writers should write this field even if it is zero or in non-null columns.
```

Comment:
```suggestion
    * Writers SHOULD always write this field even if it is zero (a.k.a. no null value)
```

Reply:
We might also add the expectation to the reader implementations?

Reply:
> We might also add the expectation to the reader implementations?

I'd like to wait other replying the context in ML firstly